README.md

```diff
@@ -101,7 +101,19 @@ $ nodemon bin/www
 
```

Comment:
Also, can you remove `#47` from the title? Makes this PR heading more confusing when trying to link to it as `#47` is actually the issue.

In your comment
> Completes the 4th task necessary for Issue #47.

That ensures that they are linked

Reply:
Gotcha, removed it now.